src/lib/device-logs/backends/redis.ts

```diff
@@ -35,16 +35,6 @@ const schema = avro.Type.forSchema({
 	],
 });
 
-export interface DeviceLog {
```

Comment:
I can't believe that we did hit this error, just like the announcement mentioned...

Reply:
Yep